autogen/cluster_regional.tf

```diff
@@ -42,10 +42,8 @@ resource "google_container_cluster" "primary" {
   logging_service    = "${var.logging_service}"
   monitoring_service = "${var.monitoring_service}"
 
-{% if private_cluster %}
```

Comment:
Let's leverage the `beta_cluster` flag, like the [provider](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/188/files#diff-a141ce951e40cb249eb91fceab0869e5R23):

```
{% if private_cluster or beta_cluster %}
```

Reply:
On second thought, the check for the private cluster is redundant in this context.

Reply:
@aaron-lane I removed the private cluster feature and just used beta.